stories/list-grid/list-grid.stories.js

```diff
@@ -0,0 +1,1880 @@
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div  class="fd-col fd-col--12 fd-col-md--6 fd-col-lg--4 fd-col-xl--3">
+                <div tabindex="0" class="fd-grid-list__item">
+                    <span class="fd-grid-list__highlight--neutral"></span>
+                    <div class="fd-grid-list__item--body fd-grid-list__item--body--padding">
+                        <div class="grid-list-item-body-example">
+                            <span class="fd-avatar fd-avatar--s grid-list-item-body-example-avatar" role="presentation" aria-label="John Doe"></span>
+                            <div>
+                                <h4 class="fd-title fd-title--h4">John Li</h4>
+                                <p>Product Owner, Company B</p>
+
+                                <div>
+                                    <p>781 Main Street</p>
+                                    <p>Anytown, SD 57401</p>
+                                    <p>USA</p>
+                                </div>
+
+                                <a href="#" class="fd-link" tabindex="0">john_li@example.com</a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+highlight.storyName = 'Highlight';
+
+highlight.parameters = {
+    docs: {
+        storyDescription: `
+A highlight indicator is shown in front of the item it needs to show that an item needs attention.
+To add highlight indicator, add a span element with \`.fd-grid-list__highlight--(success|warning|error|neutral)\` class, before the item body element
+    `
+    }
+};
+
+export const filterInfobar = () => `<div class="fd-grid-list">
+    <div class="fd-container">
+        <div class="fd-row">
+            <div class="fd-col fd-col--12">
+                <div class="fd-toolbar fd-toolbar--solid">
+                    <span>Products (420)</span>
+
+                    <span class="fd-toolbar__spacer"></span>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Sort">
+                        <i class="sap-icon--sort"></i>
+                    </button>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Filter">
+                        <i class="sap-icon--filter"></i>
+                    </button>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Group">
+                        <i class="sap-icon--group-2"></i>
+                    </button>
+                </div>
+            </div>
+        </div>
+
+        <div class="fd-row">
+            <div class="fd-col fd-col--12">
+                <div class="fd-toolbar fd-toolbar--info fd-toolbar--active">
```

Comment:
The content of this div is not announced to screen readers. The button is accessible, but simply announces 'Cancel' without context.

Reply:
fixed